src/components/registerV2/backupPassphrase.js

```diff
@@ -3,14 +3,12 @@ import QRCode from 'qrcode.react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { translate } from 'react-i18next';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-import { FontIcon } from '../fontIcon';
-import { PrimaryButtonV2, SecondaryButtonV2 } from '../toolbox/buttons/button';
+import { PrimaryButtonV2, TertiaryButtonV2 } from '../toolbox/buttons/button';
 import links from '../../constants/externalLinks';
 import Tooltip from '../toolbox/tooltip/tooltip';
+import { InputV2 } from '../toolbox/inputsV2';
 import renderPaperwallet from '../../utils/paperwallet';
-import key from '../../assets/images/icons-v2/key.svg';
-import lock from '../../assets/images/icons-v2/circle-lock.svg';
-import pdf from '../../assets/images/icons-v2/pdf.svg';
+import fileOutline from '../../assets/images/icons-v2/icon-file-outline.svg';
```

Comment:
add to `utils/svgIcons.js` file and use from there, to keep the same way we do in other places.